src/main/webapp/search_area.js

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+document.addEventListener("DOMContentLoaded", async function() {
+  // TODO: Fetch list of organizations using servlet.
+  let organizations = [{
```

Comment:
Can you make it so these organizations come from a servlet call?
const response = await fetch(`/list-organizations`);
const organizations = await response.json();

Reply:
Should I use backticks?

Reply:
I am assuming yes so that we can use the template literals for the params!